File: TenMinsJava/day77.java
// create function readCsvRow(rowId) that returns a numeric array of the row values
package TenMinsJava;

import java.io.File;
import java.util.Arrays;
import java.util.Scanner;

public class day77 {
    public static int[] readCsvRow(int rowId) {
        try {
            int currentRow = 1;
            File file = new File("D:/coding30minsAday/textFiles/JTrialCsv.csv");
            Scanner fScan = new Scanner(file);
            while (fScan.hasNextLine()) {
                String line = fScan.nextLine();
                if (currentRow == rowId) {
                    String value[]= line.split(",");
                    int row[] = new int[value.length];
                    for (int i = 0; i < value.length; i++) {
                        row[i] = Integer.parseInt(value[i].strip());
                    }
                    return row;
                }
            currentRow++;
            }
        } catch (Exception e) {
            System.err.println("Error");
        }
        return new int[0];
    }
    public static void main(String[] args) {
        int val[] = readCsvRow(2);
        System.out.println(Arrays.toString(val));
    }
}

File: TenMinsPy/day77.py
# create function readCsvRow(rowId) that returns a numeric array of the row values

import csv

class day77:
    def readCsvRow(rowId):
        with open("D:/coding30minsAday/textFiles/JTrialCsv.csv", "r", encoding="utf-8") as file:
            r = csv.reader(file)
            for i, row in enumerate(r, start=1): # only states the value one row at a time
                if i == rowId:
                    return [int(val.strip()) for val in row]
        return []

print(day77.readCsvRow(2))